src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/GerritProjectListUpdater.java

```diff
@@ -239,7 +242,8 @@ private void tryLoadProjectList() {
                         activeConfig.getGerritProxy(),
                         activeConfig.getGerritAuthentication()
                 );
-                List<String> projects = readProjects(sshConnection.executeCommandReader(GERRIT_LS_PROJECTS));
+                List<String> projects = readProjects(sshConnection
+                        .executeCommandReader(GERRIT_LS_PROJECTS + projectPrefix));
```

Comment:
Should we assume that htis works if `projectPrefix` is an empty string?
In that case I guess it won't work when it's `null`